### ISSUE TITLE
Add magic numbers?

### ISSUE THREAD

**Issue by cryptomilk** [beauwilliams/focus.nvim#21]
I have a feature in my nvim config for displaying line numbers only in the focused window:

https://git.cryptomilk.org/users/asn/dotfiles.git/tree/nvim/.config/nvim/lua/autocmds.lua#n24
https://git.cryptomilk.org/users/asn/dotfiles.git/tree/nvim/.config/nvim/lua/utils.lua#n3

We could also put this into focus.nvim

**Comment by beauwilliams** [comment]
Great idea, thought of adding it myself in the past too. I have/did have something similar in my config too, and it makes sense to be a part of focus.nvim

**Comment by cryptomilk** [comment]
I think it would be nice to be consistent with other modules and have a `setup()` function

```lua
require('focus').setup {
   enabled = true,
   ...
}
```

This could then setup all the autocmds.

**Comment by beauwilliams** [comment]
> I think it would be nice to be consistent with other modules and have a `setup()` function
> 
> ```lua
> require('focus').setup {
>    enabled = true,
>    ...
> }
> ```
> 
> This could then setup all the autocmds.

I started refactoring into this style a few minutes ago. Hopefully should not be too difficult, half way there already. I will also look at other plugins to see what the standard is too.

I chose this style earlier because it does not require any configuration i.e needing a call to setup() in the users config files (all they need to do is clone/install the package with their chosen method) but I do believe I can have the setup function there, and call it from plugin/autoload automatically so I will give it a shot soon.
